2023.md

```diff
@@ -128,7 +129,6 @@ _[dropdown]_
 - TypeScript
 - CoffeeScript
 - ClojureScript
-- Lua
```

Comment:
Lua is not JS ... it shouldn't be on this list. It can probably be transpiled, but other languages can be transpiled to JS, should be include those instead? I wouldn't think so.

Reply:
It's there because of the Defold Game Engine, which provides very compact file sizes, more comparable to Phaser etc. than Godot or Unity.

I don't see a problem including other languages, if there is some "clear path" why they got used in first place? I agree it looks a bit out of place currently.